src/mount.ts

```diff
@@ -134,8 +135,7 @@ export function mount(originalComponent: any, options?: MountingOptions) {
   }
 
   // add tracking for emitted events
-  const { emitMixin, events } = createEmitMixin()
-  vm.mixin(emitMixin)
+  attachEventListener(vm)
```

Comment:
Same here - why the change? At least imo it is much better to use the `vm.mixin` - just seeing a function with an arg and no return values screams side effects/mutation, which is rarely a good thing.

Reply:
This one is not necessary yeah. I was just trying to clean up. 

We could return the mixin and apply it here.

Reply:
That would be good! Thanks